pkg/k8s/service.go

```diff
@@ -44,14 +44,25 @@ func getAnnotationIncludeExternal(svc *slim_corev1.Service) bool {
 }
 
 func getAnnotationShared(svc *slim_corev1.Service) bool {
+	// The SharedService annotation is ignored if the service is not declared as global.
```

Comment:
just a thought, I wonder if its worth storing `Service.Shared` as separate variables. Seems like we might be overloading the meaning of shared a bit here?

Reply:
Do you mean merging the `service.IncludeExternal` and `service.Shared` variables in one? I don't think that would be possible, because `service.Shared` is ignored for the local service, while it takes effect for the remote ones.